src/main.ts

```diff
@@ -1,11 +1,17 @@
+import { ValidationPipe } from '@nestjs/common';
 import { NestFactory } from '@nestjs/core';
 
 import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
   const PORT = process.env.APP_PORT || 8000;
-
+  app.useGlobalPipes(
+    new ValidationPipe({
```

Comment:
Leave one line space after const declaration. Will add eslint config soon for linting.